### ISSUE TITLE
Minimum Size Error is raised on Mann Whitney Test with n = 30 per group

### ISSUE THREAD

**Issue by cmmorrow** [cmmorrow/sci-analysis#22]
```python
np.random.seed(987654321)
df = pd.DataFrame({'One'   : st.norm.rvs(0.0, 1, size=60),
                   'Two'   : st.norm.rvs(0.0, 3, size=60),
                   'Three' : st.weibull_max.rvs(1.2, size=60),
                   'Four'  : st.norm.rvs(0.0, 1, size=60),
                   'Month' : ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'] * 5,
                   'Condition' : ['Group A', 'Group B'] * 30})
df

analyze(df['Three'], groups=df['Condition'])
```
